java/arcs/core/storage/api/DriverAndKeyConfigurator.kt

```diff
@@ -58,8 +58,11 @@ object DriverAndKeyConfigurator {
         StorageKeyParser.reset()
 
         VolatileStorageKey.registerParser()
+        VolatileStorageKey.registerKeyCreator()
```

Comment:
registerParser() is invoked by the companion object's init{} block, so I think you can remove the lines that invoke registerParser(), and just register the key creators.